code/datums/diseases/advance/advance.dm

```diff
@@ -97,7 +97,10 @@
 		for(var/i in 1 to replace_num)
 			var/datum/disease/advance/competition = advance_diseases[i]
 			if(totalTransmittable() > competition.totalResistance())
-				competition.cure(FALSE)
+				if(!istype(competition, /datum/disease/advance/sentient_disease)) // I FUCKING SPAWNED, 30 SECONDS LATER IT WAS OVER, BECAUSE OF THIS FUCKING CODE
+					competition.cure(FALSE)
+				else
+					return FALSE // FUCK THIS MECHANIC
```

Comment:
```suggestion
			if(istype(competition, /datum/disease/advance/sentient_disease)) // I FUCKING SPAWNED, 30 SECONDS LATER IT WAS OVER, BECAUSE OF THIS FUCKING CODE
				continue
			if(totalTransmittable() > competition.totalResistance())
				competition.cure(FALSE)
			else
				return FALSE // FUCK THIS MECHANIC
```

This should make sentient diseases capable of coexisting with normal diseases.